### PR TITLE
Adding clarifications to the contributor agreement and fixing typos

### PR DESCRIPTION
**Pull request by sarah-savage**
# Pull Request

## What changed?

I clarified the legal terms in plain English, and fixed a couple typos. I also clarified (but did not change the meaning of) the section on copyleft licenses and non-copyleft repositoriies/projects.

## Why did it change?

I was asked to create a plain-English reading of the legal terms. Also, there were a few things that needed improvement/clarification and a couple typos that needed fixing.

## Did you fix any specific issues?

None.

## CERTIFICATION

By opening this pull request, I do agree to abide by the [Code of Conduct](https://github.com/aspirepress/.github/blob/updating-contributor-policy/CODE_OF_CONDUCT.md) and be bound by the terms of the [Contribution Guidelines](https://github.com/aspirepress/.github/blob/updating-contributor-policy/CONTRIBUTING.md) in effect on the date and time of my contribution as proven by the revision information in GitHub.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -63,13 +63,30 @@ We use a centralized approach that leaves `main` as the current development vers
 
 ## Licenses and Copyrights
 
+### Plain English
+
+Legal language can be tricky. Here is what our legal terms are, in plain English. Anywhere that these plain English terms conflict with the legal terms, the legal terms take precedence.
+
+* When you contribute code, you still own the code, but we are free to use it forever.
+* Contributing code to the project does not automatically mean that the end product will be suitable for a particular purpose you may have.
+* If your contribution causes us harm, and you could have prevented it, you are responsible for the harm.
```
Comment (asirota) [inline CONTRIBUTING.md:72]: This is an interesting one. I wonder if people will take on such a burden. One way to prevent the harm is not to contribute. But it's clear at least, and that I appreciate.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -63,13 +63,30 @@ We use a centralized approach that leaves `main` as the current development vers
...
+* We only allow copyleft (GPL) licensed libraries to be used in our copyleft-licensed projects. We don't allow copyleft licenses in non-copyleft projects.
```
Comment (asirota) [inline CONTRIBUTING.md:73]: So no other copy left licenses will be allowed other than GPL and its variants? Is MIT allowed?

I think it should be made clearer what copyleft licenses are. And what is not a copyleft license with examples.

**Review by asirota — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (sarah-savage) [reply CONTRIBUTING.md:72]: It's a pretty standard clause. I could modify it to specify what we're targeting here is intellectual property violations that you could have, and should have avoided. That's what the actual legalese says and I could express that kind of limit here.

File: CONTRIBUTING.md (hunk 2 above)
Comment (sarah-savage) [reply CONTRIBUTING.md:73]: GPL was meant as an example, not as an exhaustive list. I'll further clarify.

File: CONTRIBUTING.md (hunk 1 above)
Comment (asirota) [reply CONTRIBUTING.md]: Yes please. If someone leaves a project can they be retroactively liable too? Is there such a thing as open source contributors insurance?

File: CONTRIBUTING.md (hunk 2 above)
Comment (asirota) [reply CONTRIBUTING.md]: Clarify copyleft please. It's jargon and not plain language.

**Comment by chuckadams** [comment]
Looks aces to me, most of these are tacit assumptions in the FOSS world, but alas there's a reason it's called "common law" and not "common sense" 🤪 

The technical portions about coding standards are the only parts I'd still care to edit, but that should happen in another PR, keeping this one focused on explaining the legalese.

**Review by chuckadams — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -63,13 +63,30 @@ We use a centralized approach that leaves `main` as the current development vers
...
+
+Legal language can be tricky. Here is what our legal terms are, in plain English. Anywhere that these plain English terms conflict with the legal terms, the legal terms take precedence.
+
+* When you contribute code, you still own the code, but we are free to use it forever.
+* Contributing code to the project does not automatically mean that the end product will be suitable for a particular purpose you may have.
+* If your contribution causes us harm, and you could have prevented it, you are responsible for the harm. An example would be including code you know to be owned by someone else.
+* We only allow copyleft (GPL and similar) licensed libraries to be used in our copyleft-licensed projects. We don't allow copyleft licenses in non-copyleft projects.
+* We may choose a different license for any project at any time, without asking. But we won't make any open source project into a closed-source one.
+* Some users may ask us for a special license that doesn't conflict with the original license but adds new terms. This "dual licensing" is permissible.
+* If we dual license something, we won't restrict others from using the software the way it was originally inteded to be used.
```
Comment (costdev) [inline CONTRIBUTING.md:76]:
```suggestion
* If we dual license something, we won't restrict others from using the software the way it was originally intended to be used.
```

**Review by asirota — APPROVED** [review]
